src/lib/containers/download_list/CreatePackage.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react'
+import {
+  getFileHandleByIdURL,
+  getDownloadOrder,
+  getBulkFiles,
+} from '../../utils/SynapseClient'
+import {
+  BulkFileDownloadRequest,
+  ZipFileFormat,
+} from '../../utils/jsonResponses/BulkFileDownloadRequest'
+import { BulkFileDownloadResponse } from '../../utils/jsonResponses/BulkFileDownloadResponse'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import {
+  faCheck,
+  faDownload,
+  faFolder,
+} from '@fortawesome/free-solid-svg-icons'
+import * as ReactBoostrap from 'react-bootstrap'
+import { FileDownloadStatus } from '../../utils/jsonResponses/FileDownloadSummary'
+
+library.add(faCheck)
+library.add(faDownload)
+library.add(faFolder)
+
+export type CreatePackageProps = {
+  token?: string
+  children?: JSX.Element
+  updateDownloadList: Function
+}
+
+type Alert = {
+  message: string
+  variant: ReactBoostrap.AlertProps['variant']
+  className: string | undefined
+}
+
+export const templateDownload =
```

Comment:
do we have a convention to CAPITALIZE consts?

Reply:
fixed